discovery-provider/src/tasks/index.py

```diff
@@ -331,6 +331,11 @@ def fetch_cid_metadata(db, user_factory_txs, track_factory_txs, entity_manager_t
                         continue
                     if action == Action.CREATE and event_type == EntityType.USER:
                         continue
+                    if (
```

Comment:
if you want this on stage maybe add to index_nethermind too or they should share this function. it's a pain for now but we'll switch over soon.